core/controllers/topic_editor_test.py

```diff
@@ -110,21 +110,15 @@ def test_get(self):
                     ))
                 question_summary_dicts_2 = (
                     json_response['question_summary_dicts'])
-                self.assertEqual(len(question_summary_dicts_2), 1)
+                self.assertEqual(len(question_summary_dicts_2), 3)
                 self.assertEqual(
                     question_summary_dicts[0]['skill_description'],
-                    'Skill Description')
+                    ['Skill Description'])
```

Comment:
I think you should also test some case where >1 skill is attached to a question

Reply:
Done.